### PR TITLE
Add Lizy image to 404 page

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds idle Lizy image to docs 404 page.

## Motivation and Context
Finalize new 404 page updates

### CODE REVIEW

**Review by cwjohnston — APPROVED** [review]
🦎 :shipit:

File: layouts/404.html [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+    <img alt="Sensu Go Lizy mascot image" title="404 Page Not Found" src="/images/idle-lizy.png">
+
     <h1>404</h1>
+
+
+
     <h2>Page Not Found</h2>
+
+
+
```
Comment (hillaryfraley) [inline layouts/404.html]:
```suggestion
```

File: layouts/404.html [hunk 2]
```diff
@@ -1,8 +1,17 @@
 {{ partial "head" . }}
 
   <div id="page-404">
+
+    <img alt="Sensu Go Lizy mascot image" title="404 Page Not Found" src="/images/idle-lizy.png">
+
     <h1>404</h1>
+
+
+
```
Comment (hillaryfraley) [inline layouts/404.html]:
```suggestion
```

File: layouts/404.html [hunk 3]
```diff
@@ -1,8 +1,17 @@
 {{ partial "head" . }}
 
   <div id="page-404">
+
+    <img alt="Sensu Go Lizy mascot image" title="404 Page Not Found" src="/images/idle-lizy.png">
+
```
Comment (hillaryfraley) [inline layouts/404.html]:
```suggestion
```

File: layouts/404.html [hunk 4]
```diff
@@ -1,8 +1,17 @@
 {{ partial "head" . }}
 
   <div id="page-404">
+
```
Comment (hillaryfraley) [inline layouts/404.html]:
```suggestion
```

**Pull request merged**